python/ray/tests/BUILD

```diff
@@ -133,6 +132,7 @@ py_test_module_list(
 py_test_module_list(
   files = [
     "test_placement_group.py",
+    "test_object_spilling.py",
```

Comment:
This is temporary. I will make a PR to breakdown this test into two (single node vs multi node) after this PR is merged.